src/app/public/modules/popover/popover-content.component.html

```diff
@@ -12,20 +12,36 @@
   #popoverRef
 >
   <div
-    class="sky-popover sky-shadow sky-rounded-corners"
+    class="sky-popover sky-shadow"
     tabindex="-1"
+    [ngClass]="{
+      'sky-rounded-corners': themeName === 'default',
+      'sky-elevation-4': themeName === 'modern'
+    }"
   >
     <header *ngIf="popoverTitle"
-      class="sky-popover-header sky-padding-even-default"
+      class="sky-popover-header"
+      [ngClass]="{
+        'sky-padding-even-default': themeName === 'default',
+        'sky-padding-even-lg sky-margin-stacked-lg': themeName === 'modern'
+      }"
     >
       <h1 *ngIf="popoverTitle"
-        class="sky-popover-title sky-emphasized"
+        class="sky-popover-title"
+        [ngClass]="{
+          'sky-emphasized': themeName === 'default',
+          'sky-font-emphasized': themeName === 'modern'
+        }"
       >
         {{ popoverTitle }}
       </h1>
     </header>
     <div
-      class="sky-popover-body sky-padding-even-default"
+      class="sky-popover-body"
+      [ngClass]="{
+        'sky-padding-even-default': themeName === 'default',
+        'sky-padding-even-lg': themeName === 'modern'
```

Comment:
For modern theme, should we be using the `modern-` prefixed classes? For example, here we might want to use `sky-modern-theme-padding-even-lg`.

Reply:
IDK? Should we? That is a class out of the sizing class in `skyux-them` for padding. I didn't write that class.

Reply:
I think so. They are based off the modern-specific design tokens. If we decide to make a change to the modern styles, this would allow it to trickle down.

Reply:
Discussed offline and we are good here. The difference is SASS variables vs their css class equivalents.